dvc/api/artifacts.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Dict, Optional
+
+from dvc.repo import Repo
+
+
+def artifacts_show(  # noqa, pylint: disable=redefined-builtin
```

Comment:
It does not look like we are redefining a built-in here unless I am missing something.

```suggestion
def artifacts_show(  # noqa
```